tuning/config/acceleration_configs/acceleration_framework_config.py

```diff
@@ -91,13 +92,48 @@ class AccelerationFrameworkConfig:
     fast_kernels: Annotated[
         FastKernelsConfig,
         ConfigAnnotation(
-            path="peft.quantization",
+            path="training",
             key="fused_ops_and_kernels",
             experimental=True,
             required_packages=["foak"],
         ),
     ] = None
 
+    padding_free: Annotated[
+        PaddingFree,
+        ConfigAnnotation(
+            path="training.attention",
+            experimental=True,
+            required_packages=["aadp"],
+        ),
+    ] = None
+
+    multipack: Annotated[
+        MultiPack,
+        ConfigAnnotation(
+            path="training.dataloader",
+            experimental=True,
+            required_packages=["aadp"],
+        ),
+    ] = None
+
+    def _verify_configured_dataclasses(self):
+        if self.multipack is not None:
```

Comment:
What happens if multipack is set but only on single-GPU? Should there be a check to fail if a user sets this?

Reply:
we have a check in our plugin to raise https://github.com/foundation-model-stack/fms-acceleration/blob/4e81c64453ec5d2b06a8d14a2a72374cc736098a/plugins/attention-and-distributed-packing/src/fms_acceleration_aadp/framework_plugin_multipack.py#L95-L98